src/Maker/MakeCrud.php

```diff
@@ -0,0 +1,246 @@
+ */
+
+namespace Symfony\Bundle\MakerBundle\Maker;
+
+use Doctrine\Bundle\DoctrineBundle\DoctrineBundle;
+use Doctrine\Common\Inflector\Inflector;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Bundle\MakerBundle\ConsoleStyle;
+use Symfony\Bundle\MakerBundle\DependencyBuilder;
+use Symfony\Bundle\MakerBundle\Doctrine\DoctrineEntityHelper;
+use Symfony\Bundle\MakerBundle\Generator;
+use Symfony\Bundle\MakerBundle\InputConfiguration;
+use Symfony\Bundle\MakerBundle\Str;
+use Symfony\Bundle\MakerBundle\Validator;
+use Symfony\Bundle\TwigBundle\TwigBundle;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Question\Question;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Security\Csrf\CsrfTokenManager;
+use Symfony\Component\Validator\Validation;
+
+/**
+ * @author Sadicov Vladimir <sadikoff@gmail.com>
+ */
+final class MakeCrud extends AbstractMaker
+{
+    private $entityHelper;
+
+    public function __construct(DoctrineEntityHelper $entityHelper)
+    {
+        $this->entityHelper = $entityHelper;
+    }
+
+    public static function getCommandName(): string
+    {
+        return 'make:crud';
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function configureCommand(Command $command, InputConfiguration $inputConfig)
+    {
+        $command
+            ->setDescription('Creates CRUD for Doctrine entity class')
+            ->addArgument('entity-class', InputArgument::OPTIONAL, sprintf('The class name of the entity to create CRUD (e.g. <fg=yellow>%s</>)', Str::asClassName(Str::getRandomTerm())))
+            ->setHelp(file_get_contents(__DIR__.'/../Resources/help/MakeCrud.txt'))
+        ;
+
+        $inputConfig->setArgumentAsNonInteractive('entity-class');
+    }
+
+    public function interact(InputInterface $input, ConsoleStyle $io, Command $command)
+    {
+        if (null === $input->getArgument('entity-class')) {
+            $argument = $command->getDefinition()->getArgument('entity-class');
+
+            $entities = $this->entityHelper->getEntitiesForAutocomplete();
+
+            $question = new Question($argument->getDescription());
+            $question->setValidator(function ($answer) use ($entities) {return Validator::entityExists($answer, $entities); });
+            $question->setAutocompleterValues($entities);
+
+            $value = $io->askQuestion($question);
+
+            $input->setArgument('entity-class', $value);
+        }
+    }
+
+    public function generate(InputInterface $input, ConsoleStyle $io, Generator $generator)
+    {
+        $entityClassDetails = $generator->createClassNameDetails(
+            $input->getArgument('entity-class'),
+            'Entity\\'
+        );
+
+        $entityDoctrineDetails = $this->entityHelper->createDoctrineDetails($entityClassDetails->getFullName());
```

Comment:
I think we're missing some validation here to make sure that the class the user passed IS indeed a valid entity.